scripts/sequence-properties.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+const fs = require('fs');
+const regenerate = require('regenerate');
+
+const regenerateToCode = require('./utils/regenerate-to-code');
+
+const basePath = `${__dirname}/../data/sequence-properties`;
+
+const sequencePropertyNames = [
+	'Basic_Emoji',
+	'Emoji_Keycap_Sequence',
+	'RGI_Emoji_Modifier_Sequence',
+	'RGI_Emoji_Flag_Sequence',
+	'RGI_Emoji_Tag_Sequence',
+	'RGI_Emoji_ZWJ_Sequence',
+	'RGI_Emoji',
+];
+
+fs.writeFileSync(
+	`${basePath}.js`,
+	`module.exports = new Set(${JSON.stringify(sequencePropertyNames, null, 2)});`
+);
+
+fs.mkdirSync(basePath, { recursive: true });
+
+const SEP = ",";
+
+for (const propertyName of sequencePropertyNames) {
+	const data = require(`@unicode/unicode-14.0.0/Sequence_Property/${propertyName}`);
+
+	const set = regenerate();
+	const longStrings = [];
+
+	for (const str of data) {
+		if (isSingleCodePoint(str)) {
+			set.add(str.codePointAt(0));
+		} else {
+			if (str.includes(SEP)) {
+				throw new Error(`Cannot be joined with "${SEP}". Use a different separator.`);
+			}
+			longStrings.push(str);
+		}
+	}
+
+	let code = 'module.exports = {\n';
+	if (set.data.length > 0) {
+		code += `\tsingleChars: ${regenerateToCode(set, 'require("regenerate")')},\n`;
+	}
+	code += `\tlongStrings: ${JSON.stringify(longStrings.join(SEP))}.split(${JSON.stringify(SEP)})\n`;
```

Comment:
In the future there might be properties of strings that contain strings which contain whatever we choose for `SEP`. So let's just output an actual array literal?